NbMGui/src/main/java/nbmgui/Controller.java

```diff
@@ -76,6 +83,44 @@ void openFile(ActionEvent event) {
             outputView.appendText("Select a NoBeard object file\n");
     }
 
+    @FXML
+    void startProgram(ActionEvent event) {
+        setBreakpoints();
+        stepButton.setDisable(false);
+        continueButton.setDisable(false);
+        stopButton.setDisable(false);
+        Thread thread = new Thread(new Runnable() {
+            @Override
+            public void run() {
+                machine.loadStringConstants(objectFile.getStringStorage());
+                machine.loadProgram(0, objectFile.getProgram());
+                machine.runProgram(0);
+                setCurrentPcOfUI();
```

Comment:
Wouldn't a better point to call this method as soon as the machine stops because of a breakpoint?